cmd/main.go

```diff
@@ -150,7 +150,12 @@ func main() {
 
 	// dockerMetaPatchHandlers combines all Docker patch and variable handlers under a single handler.
 	// It allows to specify configuration under a single variable.
-	dockerMetaPatchHandlers := []mutation.MetaMutater{}
+	dockerMetaPatchHandlers := append(
+		[]mutation.MetaMutater{
+			region.NewMetaPatch(),
```

Comment:
We don't need this, aws specific.

Reply:
Oops copy and paste without really looking, fixing 🤦‍♂️